packages/builders/scripts/orchestration/init-stakeAtom.js

```diff
@@ -11,7 +11,7 @@ export const defaultProposalBuilder = async (
     bondDenom = 'uatom',
   } = options;
   return harden({
-    sourceSpec: '@agoric/orchestration/src/proposals/start-stakeAtom.js',
+    sourceSpec: '@agoric/orchestration/src/core/start-stakeAtom.js',
```

Comment:
did you consider renaming `start-stakeAtom.js` to `stakeAtom.deploy.js` as well?

Reply:
ooh, I like that. so a `.deploy.js` is a module that exports a manifest and its referenced functions for use in a build script? I'm reluctant do to that since the module shape isn't uniform. Every export name changes from one to the next, even if certain exports are supposed to fit a pattern. Maybe if we had a new deploy module format that had a `export const meta` for looking things up.

Regardless, I consider all that out of scope for clarifying CoreEval vs Proposal.